Sources/MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -189,7 +189,7 @@ public class CarPlayMapViewController: UIViewController {
     func setupNavigationMapView() {
         let navigationMapView = NavigationMapView(frame: UIScreen.main.bounds, navigationCameraType: .carPlay)
         navigationMapView.delegate = self
-        navigationMapView.mapView.mapboxMap.onNext(.styleLoaded) { _ in
+        navigationMapView.mapView.mapboxMap.onEvery(.styleLoaded) { _ in
```

Comment:
Generally speaking, StyleManager is responsible for applying a new style initially and whenever switching between DayStyle and NightStyle. We can’t use StyleManagerDelegate methods directly, because these operations need to wait until the style has finished loading, or else they won’t have any effect.

Reply:
Disabling traffic on Night/Day looks unexpected to me.

Reply:
Traffic was disabled in #3209. This change just keeps it off if the user happens to be using CarPlay around sunrise or sunset.

@MaximAlien, is there additional context as to why users should not see traffic during browsing and previewing? These are the situations where we used to use the Navigation Preview Day/Night styles. NavigationMapView may not show traffic by default, but that’s because it isn’t necessarily being used as a preview map; the Example application’s ViewController should enable it in that case.